packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2956,6 +2972,22 @@ class InputDecoration {
   /// Only one of [error] and [errorText] can be specified.
   final String? errorText;
 
+  /// Builds the [Widget] that appears below the [InputDecorator.child] and the border.
+  ///
+  /// If non-null, [errorText] will be passed to this builder, and the returned
+  /// widget will be displayed below the [InputDecorator.child] and the border.
```

Comment:
Similar to my comment about the assertion, I would add: "[errorText] must be non-null if [errorBuilder] is non-null."

Reply:
This is not the case since errorBuilder will always be there, in most cases shared bw multiple fields, and errorText comes from the validator, and whether or not it exists depends on the validation of a specific field.